### PR TITLE
Use POST and provide success and error pages

### PR DESCRIPTION
**Pull request by Demotive**

### CODE REVIEW

File: public/js/app.js [hunk 1]
```diff
@@ -231,7 +231,26 @@ var runExperiment = function() {
   return x();
 };
 
+// Run the experiment then post to server
 runExperiment().then(function() {
```
Comment (bottlerocket) [inline public/js/app.js]: Should we decouple the function here, so it is easier to read, as well being more modular.

File: server.js [hunk 2]
```diff
@@ -1,9 +1,11 @@
 var express = require('express');
 var app = express();
-
+var bodyParser = require('body-parser');
 var json2csv = require('json2csv');
 var fs = require('fs');
 
+app.use(bodyParser.json());
+
 var csvFields = ['correct', 'duration', 'real', 'response', 'responseTime', 'state', 'word'];
```
Comment (bottlerocket) [inline server.js]: if we add additional items to this array, will this automatically update the fields it outputs or does JSON2CSV have some hard coded stuff in it that we need to update as well. i.e. just a comment. showing where to make changes to the JSON you're updating.

File: public/js/app.js (hunk 1 above)
Comment (Demotive) [inline public/js/app.js]: Yes we should. I am sorry Tim.

**Comment by bottlerocket** [comment]
looks fine, according to my 'expert' eyes.

**Pull request merged**